test/metabase/db/connection_test.clj

```diff
@@ -16,31 +19,31 @@
   (fixtures/initialize :db))
 
 (deftest nested-transaction-test
-  (let [user-1                    (mt/random-email)
```

Comment:
The indentation in this block was fine before. Generally in a `let` form (or values in a map) it's OK to align the values (i.e. the original code here) or use a single space for each one (i.e. what you've changed it to), as long as it's consistent within the form.

So don't feel the need to revert, but the original was OK too. If your editor changed this automatically I'd look into configuring it or else it'll be auto-correcting a lot of code.